third_party/txt/src/txt/text_shadow.cc

```diff
@@ -41,7 +41,7 @@ bool TextShadow::operator!=(const TextShadow& other) const {
 bool TextShadow::hasShadow() const {
   if (!offset.isZero())
     return true;
-  if (blur_radius != 0.0)
+  if (blur_sigma != 0.0)
```

Comment:
We have an issue here. A 0 radius does not convert into a 0 sigma, so the test for 0 no longer works.

One thing I notice is that the dart conversion function used above says that it is based on the Skia version and provides a link to the Skia conversion code. If you take a look at the Skia equations, they make sure that a radius of 0 produces a sigma of 0 and that a sigma of less than 0.5 produces a radius of 0.

The net effect should be that we might want to update our Dart conversion function to match the 0 => 0 behavior of the Skia function and to change this test to `blur_sigma <= 0.5`?

Reply:
make sence, thanks!   the test should be "blur_sigma > 0.5" which means it is a valid value. right?

Reply:
Right, `> 0.5` means it will do something. I got that backwards.